package.json

```diff
@@ -29,7 +29,7 @@
     "axios-middleware": "^0.3.1",
     "cookies-next": "^2.1.1",
     "lodash": "^4.17.21",
-    "next": "12.3.1",
+    "next": "^13.2.5-canary.30",
```

Comment:
Introduced a breaking change that allows us to directly style links without weird children structure. See Upgrade Guide: https://beta.nextjs.org/docs/upgrade-guide#link-component